PocketKit/Sources/PocketKit/Home/HomeViewModel.swift

```diff
@@ -98,17 +98,27 @@ class HomeViewModel {
 
     private let source: Source
     private let tracker: Tracker
+    private let networkPathMonitor: NetworkPathMonitor
     private var subscriptions: [AnyCancellable] = []
     private var recentSavesCount: Int = 0
 
     init(
         source: Source,
-        tracker: Tracker
+        tracker: Tracker,
+        networkPathMonitor: NetworkPathMonitor
     ) {
         self.source = source
         self.tracker = tracker
+        self.networkPathMonitor = networkPathMonitor
 
         self.snapshot = {
+            guard networkPathMonitor.currentNetworkPath.status == .satisfied else {
+                var snapshot = Snapshot()
+                snapshot.appendSections([.offline])
+                snapshot.appendItems([.offline], toSection: .offline)
+                return snapshot
```

Comment:
when I log in, I see the offline view for a bit even though I have internet connection. Seems like the network status is `Network.NWPath.Status.unsatisfied` at that point

Reply:
I'll take a peek 👀

Reply:
looks good now!